target_redshift/db_sync.py

```diff
@@ -381,7 +383,8 @@ def load_csv(self, s3_key, count):
                     self.connection_config['s3_bucket'],
                     s3_key,
                     self.connection_config['aws_access_key_id'],
-                    self.connection_config['aws_secret_access_key']
+                    self.connection_config['aws_secret_access_key'],
+                    "SESSION_TOKEN '{}'".format(self.connection_config['aws_session_token']) if self.connection_config['aws_session_token'] else ''
```

Comment:
it gives KeyError: 'aws_session_token' when aws_session_token is not defined in config.json

Can we change and shorten it to `"SESSION_TOKEN '{}'".format(self.connection_config.get('aws_session_token', ''))` ?

Reply:
thanks for fixing it by 29f76d5 .
Similar problem above is still exist and throws `KeyError: 'aws_session_token'` when initialising boto3